docs/wiki/CL/cl-clients.md

```diff
@@ -1,178 +1,104 @@
-# Consensus Layer Implementations
+# Consensus Client
 
-This page covers resources on all consensus client implementations, whether in production or development. It provides an overview of unique features of each client, architecture, basic guides and resources.
+> **Consensus clients**, formerly known as *eth2 clients*, run Ethereum's proof-of-stake consensus algorithm allowing the network to reach agreement about the head of the Beacon Chain. Consensus clients do not participate in validating/broadcasting transactions or executing state transitions: that is done by execution clients. Consensus clients do not attest to, or propose new blocks: that is done by the validator client which is an optional add-on to the consensus client.
 
-> Consensus clients originally used to be called _eth2.0 clients_ which is now a deprecated nomenclature but you can still find this reference in their repositories.
 
-There are multiple Consensus Layer clients developed to participate in the Ethereum Proof-of-Stake (PoS) mechanism. The most popular, FOSS and production ready are [Lighthouse](https://lighthouse-book.sigmaprime.io/), [Lodestar](https://lodestar.chainsafe.io/), [Nimbus](https://nimbus.team/index.html), [Prysm](https://prysmaticlabs.com/) and [Teku](https://consensys.io/teku). These clients are developed in different programming languages, provide have unique features and offer different performance profiles. All clients support Ethereum mainnet out of the box along with active testnets. Variety of implementations allows the network to benefit from client diversity. If you are choosing a client to use, current client diversity should be one of the main factors.
+## Overview Table
 
-## Clients in production
+| Client                                                                  | Language   | Developer           | Status      |
+| ----------------------------------------------------------------------- | ---------- | ------------------- | ----------- |
+| [Lighthouse](https://github.com/sigp/lighthouse)                        | Rust       | Sigma Prime         | Production  |
+| [Lodestar](https://github.com/ChainSafe/lodestar)                       | TypeScript | ChainSafe           | Production  |
+| [Nimbus](https://github.com/status-im/nimbus-eth2)                      | Nim        | Status              | Production  |
+| [Prysm](https://github.com/prysmaticlabs/prysm)                         | Go         | Prysmatic Labs      | Production  |
+| [Teku](https://github.com/ConsenSys/teku)                               | Java       | ConsenSys           | Production  |
+| [Grandine](https://github.com/grandinetech/grandine)                    | Rust       | Grandine Developers | Production  |
+| [Caplin](https://github.com/ledgerwatch/erigon)                         | N/A        | Erigon              | Development |
```

Comment:
I suppose this should say "Go" under Language for Caplin

Reply:
of course